app/src/androidTest/java/org/mozilla/fenix/ui/robots/CollectionRobot.kt

```diff
@@ -289,3 +294,8 @@ private fun backButton() =
     mDevice.findObject(
         UiSelector().resourceId("$packageName:id/back_button"),
     )
+private val addCollectionButtonPanel =
+    itemWithResId("$packageName:id/buttonPanel")
+
+private val addCollectionOkButton =
+    itemWithResId("android:id/button1")
```

Comment:
Is it possible to use `Android.R` here?

Reply:
Not sure, will try in a different PR and update in all robots if it works.